layer/json/VkLayer_gfxreconstruct.json.in

```diff
@@ -290,6 +275,23 @@
                                     }
                                 ]
                             }
+                        },
+                        {
+                            "key": "page_guard_persistent_memory",
```

Comment:
This option has existed for a while but was only recently documented

Reply:
I think you can update the dependence section to take into account the `GFXRECON_PAGE_GUARD_EXTERNAL_MEMORY` dependence by:
```
                        {
                            "key": "page_guard_persistent_memory",
                            "env": "GFXRECON_PAGE_GUARD_PERSISTENT_MEMORY",
                            "label": "Page Guard Persisent Memory",
                            "description": "When the page_guard memory tracking mode is enabled, this option changes the way that the shadow memory used to detect modifications to mapped memory is allocated. The default behavior is to allocate and copy the mapped memory range on map and free the allocation on unmap. When this option is enabled, an allocation with a size equal to that of the object being mapped is made once on the first map and is not freed until the object is destroyed. This option is intended to be used with applications that frequently map and unmap large memory ranges, to avoid frequent allocation and copy operations that can have a negative impact on performance. This option is ignored when GFXRECON_PAGE_GUARD_EXTERNAL_MEMORY is enabled.",
                            "type": "BOOL",
                            "default": false,
                            "dependence": {
                                "mode": "ALL",
                                "settings": [
                                    {
                                        "key": "memory_tracking_mode",
                                        "value": "page_guard"
                                    },
                                    {
                                        "key": "page_guard_external_memory",
                                        "value": false
                                    }
                                ]
                            }
                        }
                        ```